java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -95,7 +95,7 @@ public ActivateTrainFrame(DispatcherFrame d) {
     private final JComboBox<String> destinationBlockBox = new JComboBox<>();
     private List<Block> destinationBlockBoxList = new ArrayList<>();
     private List<Integer> destinationBlockSeqList = new ArrayList<>();
-    private JButton addNewTrainButton = null;
+    public JButton addNewTrainButton = null;
```

Comment:
If it's desired to be able to click on the button from a script, it's usually better to add a method `clickAddNewTrainButton()` instead of making the variable public.

Reply:
I think I can put this back to private. Sorry, another failed edit when I was trying to solve a fault.